plugins/lang-plan-a/src/main/antlr/PlanAParser.g4

```diff
@@ -34,18 +34,23 @@ statement
     | CONTINUE SEMICOLON?                                                                    # continue
     | BREAK SEMICOLON?                                                                       # break
     | RETURN expression SEMICOLON?                                                           # return
-    | TRY block ( CATCH LP ( TYPE ID ) RP block )+                                           # try
+    | TRY block trap+                                                                        # try
     | THROW expression SEMICOLON?                                                            # throw
     | expression SEMICOLON?                                                                  # expr
     ;
 
 block
-    : LBRACK statement* RBRACK                 # multiple
+    : LBRACK statement+ RBRACK                 # multiple
```

Comment:
I have separated out the two because in some cases it doesn't make sense to have an empty statement, and it's easy to let the grammar error out rather than handle that later on in the code.  I realize that in most languages you can do something like "if (x) {}", but there there is really no point in allowing this in Plan A.